rozdzial1.tex

```diff
@@ -22,10 +20,9 @@ \subsection{Jakiś tytuł 2}
 
 %---------------------------------------------------------------------------
 
-\section{Zawartość pracy}
-\label{sec:zawartoscPracy}
+\section{Zawartość pracy}\label{sec:zawartoscPracy}
 
-W rodziale~\ref{cha:pierwszyDokument} przedstawiono podstawowe informacje dotyczące struktury dokumentów w \LaTeX u. Alvis~\cite{Alvis2011} jest językiem 
+W rodziale~\ref{cha:pierwszyDokument} przedstawiono podstawowe informacje dotyczące struktury dokumentów w \LaTeX{}u. Alvis~\cite{Alvis2011} jest językiem 
```

Comment:
Możesz mi powiedzieć jaka jest zaleta dodawania pustej grupy `{}` po makrze `\LaTeX`? Potencjalnie może to psuć kerning pomiędzy iksem z LaTeX a kolejną literą (np. XO).

Reply:
Nie jestem miestrzem LaTeXa więc korzystam tylko i wyłącznie z tego, że `\LaTeX `(ze spacją na końcu) powoduje ostrzeżenie. Rozumiem czemu należy użyć `\LaTeX\ ` (jeśli chcemy spację) ale trzeba też jakoś wymusić połączenie literki `u` z `\LaTeX`. Najprostrze wydaje się więc składnia przypomianająca wywołanie funkcji bez argumentów. Jeśli masz jednak inną propozycję chętnie zamienię.

Reply:
Według dokumentu z wytłumaczeniem ostrzeżeń to ostrzeżenie jest po to by nie zapomnieć o zjadaniu spacji przez makra, gdyż zazwyczaj po takim makrze chce się spacje. W przypadkach które poprawiłeś nie chcemy spacji więc wydaje mi się że oryginalna wersja tzn. np. `\LaTeX a` to najlepsze rozwiązanie, które nie wprowadza potencjalnych problemów.

Reply:
No to w sumie jest to słabe - bo jeśli zaczynają się pojawiać ostrzeżenia, które ignorujemy - to powoduje, że między nimi może umknąć istotne. Widziałem, że da się je wyłączać - wyłączę więc je dla tego konkretnego przypadku.